e2e/e2e_test.go

```diff
@@ -26,27 +27,33 @@ var (
 )
 
 type testConfig struct {
-	TestNamservers            []string
+	TestNameservers           []string
 	Kubeconfig                string
 	CertID, CertVersionlessID string
 	DNSZoneDomain             string
 }
 
 func TestMain(m *testing.M) {
 	// Load configuration
-	rawConf, err := ioutil.ReadFile("../devenv/state/e2e.json")
-	if err != nil {
-		panic(err)
+	rawConf := os.Getenv("E2E_JSON_CONTENTS")
+	if rawConf == "" {
+		panic(fmt.Errorf("failed to get e2e contents from env"))
```

Comment:
use errors.New instead of fmt.Errorf when you don't need to format any variables in

Reply:
resolved but not updated?

Reply:
ah was on an old version of the pr all good